### PR TITLE
core/job: emit job start message if we're only waiting for unit state

### PR DESCRIPTION
**Pull request by YHNdnzj**

### CODE REVIEW

File: src/core/job.c [hunk 1]
```diff
@@ -853,28 +852,33 @@ static int job_perform_on_unit(Job **j) {
         switch (t) {
                 case JOB_START:
                         r = unit_start(u, a);
+                        wait_only = r == -EBADF; /* If the unit type does not support starting, then simply wait. */
```
Comment (dtardon) [inline src/core/job.c]: `EBADF` -> `EBADR`?

File: src/core/job.c [hunk 2]
```diff
@@ -853,28 +852,33 @@ static int job_perform_on_unit(Job **j) {
...
                         break;
 
                 case JOB_RESTART:
                         t = JOB_STOP;
                         _fallthrough_;
                 case JOB_STOP:
                         r = unit_stop(u);
+                        wait_only = r == -EBADF; /* If the unit type does not support stopping, then simply wait. */
```
Comment (dtardon) [inline src/core/job.c]: Also here.

File: src/core/job.c (hunk 1 above)
Comment (YHNdnzj) [reply src/core/job.c]: Oops, thank you

**Review by bluca — APPROVED** [review]

**Review by yuwata — APPROVED** [review]
LGTM.

**Pull request merged**